boards/arm/imx6/sabre-6quad/scripts/Make.defs

```diff
@@ -39,8 +39,8 @@ ifneq ($(CONFIG_DEBUG_NOOPT),y)
 endif
 
 ARCHCPUFLAGS = -mcpu=cortex-a9 -mfpu=vfpv4-d16
-ARCHCFLAGS = -fno-builtin -ffunction-sections -fdata-sections
-ARCHCXXFLAGS = -fno-builtin -fno-exceptions -fcheck-new -fno-rtti
+ARCHCFLAGS = -fno-builtin -ffunction-sections -fdata-sections -fno-common
```

Comment:
After checking other Make.defs, it look like the better place is add -fno-common to CELFFLAGS/CXXELFFLAGS.

Reply:
> After checking other Make.defs, it look like the better place is add -fno-common to CELFFLAGS/CXXELFFLAGS.

@xiaoxiang781216 
I found that adding -fno-common to CELFFLAGS/CXXELFFLAGS does not work with CONFIG_BUILD_KERNEL=y.

Reply:
If so, should we fix all place?

Reply:
@xiaoxiang781216 
I found that it's needed for libraries such as libc.a which is included in nuttx-export-x.x.x.zip and used to link applications.

Reply:
It look the better fix is add -fno-common to all ARCHCPUFLAGS/ARCHCXXFLAGS and remove from CELFFLAGS/CXXELFFLAGS.